bench/workgen/runner/runner/__init__.py

```diff
@@ -27,7 +27,7 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 # runner/__init__.py
-#	Used as a first import by runners, does any common initialization.
+#   Used as a first import by runners, does any common initialization.
```

Comment:
I needed to remove this tab because __init__.py is no longer a generated file anywhere in the tree.